From 1ee059ed47b379c1cd9494c507a4a1e67dc90cf2 Mon Sep 17 00:00:00 2001
From: Eric Ruei <e-ruei1@ti.com>
Date: Mon, 12 Mar 2018 17:56:50 -0400
Subject: [PATCH 1/2] pvr_drm.c: fix undeclared identifier error for AM3/4
 builds

The undeclared identifier error was introduced during the k4.14 migration
and therefore was not applicable to the k4.4 and k4.9 branches.

Signed-off-by: Eric Ruei <e-ruei1@ti.com>
---
 eurasia_km/services4/srvkm/env/linux/pvr_drm.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/eurasia_km/services4/srvkm/env/linux/pvr_drm.c b/eurasia_km/services4/srvkm/env/linux/pvr_drm.c
index 39ee832..33f6c3d 100644
--- a/eurasia_km/services4/srvkm/env/linux/pvr_drm.c
+++ b/eurasia_km/services4/srvkm/env/linux/pvr_drm.c
@@ -655,12 +655,12 @@ static struct platform_driver sPVRPlatDriver =
 static int
 PVRSRVDrmProbe(struct platform_device *pDevice)
 {
-#if (AM_VERSION != 5)
-	struct gfx_sgx_platform_data *pdata = dev->platform_data;
-#endif
 	int ret;
 	struct device *dev = &pDevice->dev;
 	struct drm_device *drm_dev;
+#if (AM_VERSION != 5)
+	struct gfx_sgx_platform_data *pdata = dev->platform_data;
+#endif
 
 	PVR_TRACE(("PVRSRVDrmProbe"));
 
-- 
1.9.1

